packages/smart-accounts/src/clients.ts

```diff
@@ -0,0 +1,45 @@
+import { createBundlerClient } from "viem/account-abstraction";
+import type { Chain, Transport } from "viem";
+import type { SmartAccount } from "viem/account-abstraction";
+import {
+  alchemyEstimateFeesPerGas,
+  alchemyRpcSchema,
+} from "./alchemyEstimateFeesPerGas.js";
+
+/**
+ * Create a viem Bundler Client pre-configured for Alchemy Bundler:
+ *
+ * • Injects `rundler_maxPriorityFeePerGas` fee estimation via
+ *   {@link alchemyEstimateFeesPerGas}.
+ * • Extends the RPC schema with {@link alchemyRpcSchema} so the new method is
+ *   typed.
+ *
+ * @param {object} params                    – configuration object.
+ * @param {SmartAccount} params.account      – Smart account instance for the client.
+ * @param {Chain | undefined} params.chain   – Chain the bundler operates on.
+ * @param {Transport} params.transport       – viem transport (eg. `custom(client)`).
+ * @returns {ReturnType<typeof createBundlerClient>} a Bundler Client ready to use with Alchemy’s Rundler.
+ */
+export function createAlchemyBundlerClient<
```

Comment:
This is fine to keep for now, but I'm not sure we need a `AlchemyBundlerClient` - it may be sufficient to tell users to use `createBundlerClient` with the `alchemyFeeEstimator`. Could you leave a `TODO(v5)` commet to revisit this export?